app/services/trading_strategies.py

```
@@ -4153,6 +4153,405 @@ def _calculate_portfolio_sortino(self, positions: List[Dict]) -> float:
+                    "type": "funding_threshold",
+                    "condition": f"funding_rate > {funding_rate * 1.2:.6f}",
+                    "rationale": "Enter short when funding is expensive"
+                })
+                
+            conditions.append({
+                "type": "liquidity_check",
+                "condition": "24h_volume > $10M",
+                "rationale": "Ensure sufficient liquidity"
+            })
+                
+            return conditions
```

Ruff (0.12.2)

Consider moving this statement to an `else` block

(TRY300)